readme.md

```diff
@@ -1,8 +1,6 @@
 # Smaily for Opencart
 
-> DISCLAIMER Plugin is still in development and currently being reviewed by Opencart. Use at your own discretion before fully finished. All bug reports are welcome under issues tab.
-
-Simple and flexible Smaily integration for Opencart.
+> Simple and flexible Smaily integration for Opencart.
```

Comment:
Remove `>`. No need to make this sentence a blockquote.

Reply:
Fixed https://github.com/sendsmaily/smaily-opencart-module/pull/1/commits/5bedbd12ccd5810dcc735616738eab07290ce006